extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -54,33 +50,40 @@ export class AmpStoryRequestService {
   }
 
   /**
-   * Retrieves the publisher share providers.
-   * Has to be called through `loadShareConfig`.
+   * Retrieves the inline config - will be called if
+   * src attribute is invalid or not present.
+   * @param  {!Element} configEl
```

Comment:
Plz be consistent with the naming, you pass `element` and get `configEl` when there is no clear reason for having two names referring to the same thing within the same file